Containerfile

```diff
@@ -17,11 +15,12 @@ COPY scripts/download_embeddings_model.py .
 RUN pdm run python download_embeddings_model.py -l ./embeddings_model -r ${EMBEDDING_MODEL}
 
 COPY scripts/generate_embeddings.py .
-RUN pdm run python generate_embeddings.py -f ocp-product-docs-plaintext/${OCP_VERSION} -md embeddings_model \
-    -mn ${EMBEDDING_MODEL} -o vector_db/ocp_product_docs/${OCP_VERSION} \
-    -i ocp-product-docs-$(echo $OCP_VERSION | sed 's/\./_/g') -v ${OCP_VERSION}
+RUN for OCP_VERSION in 4.15 4.16; do \
```

Comment:
would be even better if this just processed all the OCP versions in a given directory, so you don't have to update it each time you add a new version.  Not that we expect this to be the long term approach anyway so it doesn't matter a whole lot.